### PR TITLE
docs/mixin: jsonnetify mixin to allow using common mixin tooling

### PR DESCRIPTION
**Pull request by paulfantom**
## Description

_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue._

This PR is making `docs/mixin` directory a self-contained package. It is also adding jsonnet wrapper around mixin and improving documentation

## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [ ] CHANGELOG entry for user-facing changes
- [ ] Updated the relevant documentation

### CODE REVIEW

**Comment by paulfantom** [comment]
Rebased to fix merge conflicts

**Review by cevian — COMMENTED** [review]
Can you expand on which tooling this change allows us to use? Why do we want jsonnet?

**Comment by paulfantom** [comment]
This allows us to:
- put the mixin on https://monitoring.mixins.dev/
- easily synchronize mixin in tobs
- use `mixtool` (a tool for managing mixins)
- potentially allow customization of mixin itself in the future

**Comment by paulfantom** [comment]
This PR is also regenerating alerts in helm chart and moving CI check for alerts propagation into helm workflow. The latter is done because that check is not about mixin itself, but about helm chart.

**Review by cevian — APPROVED** [review]

**Comment by paulfantom** [comment]
Argh, wrong button...

**Comment by paulfantom** [comment]
rebased

**Comment by paulfantom** [comment]
rebased and regenerated

File: deploy/helm-chart/templates/prometheus-rule.yaml [hunk 1]
```diff
@@ -229,6 +240,7 @@ spec:
       annotations:
         summary: Promscale experiences a high error rate when connecting to the database
         description: "Promscale connection with the database has an error of {{ $value | humanizePercentage }}."
+        runbook_url: https://github.com/timescale/promscale/blob/master/docs/runbooks/PromscaleStorageHighErrorRate.md
```
Comment (Harkishen-Singh) [inline deploy/helm-chart/templates/prometheus-rule.yaml:243]: This is now 
```suggestion
        runbook_url: https://github.com/timescale/promscale/blob/master/docs/runbooks/PromscaleDBHighErrorRate.md
```

**Review by Harkishen-Singh — APPROVED** [review]
Didn't go deep but looks good to merge.

**Pull request merged**